test/driver.js

```diff
@@ -278,6 +265,67 @@ var rasterizeAnnotationLayer = (function rasterizeAnnotationLayerClosure() {
   return rasterizeAnnotationLayer;
 })();
 
+/**
+ * @class
+ */
+var rasterizeXfaLayer = (function rasterizeXfaLayerClosure() {
+  const styles = {
+    common: {
+      file: "../web/xfa_layer_builder.css",
+      promise: null,
+    },
+  };
+
+  function getXfaLayerStyle() {
+    return loadStyles(styles);
+  }
+
+  // eslint-disable-next-line no-shadow
+  function rasterizeXfaLayer(ctx, viewport, xfa, fontRules) {
+    return new Promise(function (resolve, reject) {
+      // Building SVG with size of the viewport.
+      const svg = document.createElementNS(SVG_NS, "svg:svg");
+      svg.setAttribute("width", viewport.width + "px");
+      svg.setAttribute("height", viewport.height + "px");
+
+      // Adding element to host our HTML (style + text layer div).
```

Comment:
Nit: Too much copy-and-paste here :-)
`// Adding element to host our HTML (style + xfa layer div).`